app/views/list/index.html.slim

```diff
@@ -0,0 +1,30 @@
+.container-fluid
+  .row
+    .col-xs-12
+      .wishlist-header
+        | Wishlists
+      .wishlist-page
+        - if @lists.present?
+          .table-responsive
+            table#catalog-table.wishlist-table.table.table-hover
+              thead
+                th
+                th
+                th
+                  | Title
+                th
+                  | Description
+                th
+                  | Created At
+                th
+                  | Updated At
+              tbody
+                - @lists.each do |list|
```

Comment:
you can just do `= render partial: 'list/list_row_template', collection: lists`, see here: http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections